apps/docs/src/app/core/component-docs/truncate/examples/truncate-example.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component } from '@angular/core';
```

Comment:
shouldn't all the files related to the truncate directive be removed? as now we have pipe that we don't expose to the customers

Reply:
Directive is still there truncating elements by width. So it is example for that. If that is the case for pipe, I can remove pipe-examples.